src/bundles.ts

```diff
@@ -209,12 +209,14 @@ export async function getSupportedFiles(
   const supportedFilesFromApi = resp.value;
   //Given supported languages from 'registy'
   if (languages) {
-    const supportedFiles: SupportedFiles = {} as any;
+    const supportedFiles = {} as SupportedFiles;
```

Comment:
enh: Couldn't we initialize this directly as:

```ts
const supportedFiles: SupportedFiles = {
  configFiles: supportedFilesFromApi.configFiles,
  extensions: languages
}
```

This would improve the type safety, simplify the logic, and remove the cast.